pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -286,6 +287,89 @@ func (a *ofFlowAction) Group(id GroupIDType) FlowBuilder {
 	return a.builder
 }
 
+//  Learn is an action adds or modifies a flow in an OpenFlow table.
+func (a *ofFlowAction) Learn(id TableIDType, priority uint16, idleTimeout uint16, cookie uint64) LearnAction {
+	la := &ofLearnAction{
+		flowBuilder: a.builder,
+		nxLearn:     ofctrl.NewLearnAction(uint8(id), priority, idleTimeout, 0, 0, 0, cookie),
+	}
+	la.nxLearn.DeleteLearnedFlowsAfterDeletion()
+	return la
+}
+
+// ofLearnAction is used to describe actions in the learn flow.
+type ofLearnAction struct {
+	flowBuilder *ofFlowBuilder
+	nxLearn     *ofctrl.FlowLearn
+}
+
+// MatchLearntTCPDSTPort makes the learnt flow to match the tp_dst of current TCP packet.
+func (a *ofLearnAction) MatchLearntTCPDSTPort() LearnAction {
+	ethTypeVal := make([]byte, 2)
+	binary.BigEndian.PutUint16(ethTypeVal, 0x800)
+	ipTypeVal := make([]byte, 2)
+	ipTypeVal[1] = byte(ofctrl.IP_PROTO_TCP)
+	a.nxLearn.AddMatch(&ofctrl.LearnField{Name: "NXM_OF_ETH_TYPE"}, 2*8, nil, ethTypeVal)
+	a.nxLearn.AddMatch(&ofctrl.LearnField{Name: "NXM_OF_IP_PROTO"}, 1*8, nil, ipTypeVal)
+	a.nxLearn.AddMatch(&ofctrl.LearnField{Name: "NXM_OF_TCP_DST"}, 2*8, &ofctrl.LearnField{Name: "NXM_OF_TCP_DST"}, nil)
+	return a
+}
+
+// MatchLearntTCPDSTPort makes the learnt flow to match the tp_dst of current UDP packet.
```

Comment:
you copied `MatchLearntTCPDSTPort` everywhere instead of using the actual method name in the comment